echoscu/src/main.rs

```diff
@@ -3,7 +3,7 @@ use dicom_core::{dicom_value, DataElement, VR};
 use dicom_dictionary_std::{tags, uids};
 use dicom_object::{mem::InMemDicomObject, StandardDataDictionary};
 use dicom_ul::{
-    association::client::ClientAssociationOptions,
+    association::{client::ClientAssociationOptions, Association, SyncAssociation},
```

Comment:
Wanted to get your thoughts on this too, its not ideal to need to import two traits, but I didn't know a better way to do it